deployment/ccip/changeset/state.go

```diff
@@ -312,7 +364,11 @@ func (s CCIPOnChainState) View(chains []uint64) (map[string]view.ChainView, erro
 		if err != nil {
 			return m, err
 		}
-		m[chainInfo.ChainName] = chainView
+		name := chainInfo.ChainName
+		if chainInfo.ChainName == "" {
```

Comment:
should never happen right?

Reply:
happens for memory env, all the testnets have blank name